gamebook/game_manager.py

```diff
@@ -0,0 +1,42 @@
+from gamebook.terminal_output import TerminalOutput
+from gamebook.terminal_input import TerminalInput
+import sys
+
+
+class GameManager(object):
+    def __init__(self, scene):
+        self.scene = scene
+        self.output_instance = TerminalOutput()
+        self.input_instance = TerminalInput()
+
+    def start(self, scene_import):
+        while True:
+            next_scene_name = self.run_scene()
```

Comment:
Why can't the scene return the next Scene Object? and not just a name? 
ie, import the next scene in the scene itself, that way you don't need to do a string search and also add the imports of the scenes, right?

Reply:
Unless it seems more elegant this way.

Reply:
i thought of changing the entire way that the next_scene parameter in `Scene` will work so that the user will put the name of the next_scene instead of the `NextScene` type, then when he will build the scenes for the game he wouldn't have to deal with `Using variable  before assignment` issues, and have a more organized and chronological order to his scenes

Reply:
Do you have examples of each approach for the user to show?

Reply:
couldn't make one yet, because it breaks other functions.
i don't think i could finish refactoring everything in time, but for now i want GameManager to already work this way so it will be easier to change the code later